packages/react-core/src/components/Dropdown/examples/Dropdown.md

```diff
@@ -18,133 +18,159 @@ import avatarImg from '../../Avatar/examples/avatarImg.svg';
+
+```ts file="./DropdownPlainTextToggle.tsx"
 ```
 
-### Alignments on different breakpoints
+### Disabled toggles
 
-```ts file="./DropdownAlignmentOnBreakpoints.tsx"
+A `<DropdownToggle>` may use the `isDisabled` property to disable the expansion of the dropdown menu.
+
+```ts file="./DropdownDisabled.tsx"
 ```
 
-### Direction up
+### Icon toggle
 
-```ts file="./DropdownDirectionUp.tsx"
+`<DropdownToggle>` may use a recognizable icon instead of a text label if text would take up too much room. Set the `toggleIndicator` property to `{null}` to remove the default dropdown toggle caret.
+
+```ts file="./DropdownIconOnly.tsx"
 ```
 
-### With kebab
+### Kebab toggle
+
+Use `<KebabToggle>` to replace the default text label and caret with a kebab icon. A kebab toggle can be used to indicate that a dropdown connects to a menu.
 
 ```ts file="./DropdownKebab.tsx"
 ```
 
-### With badge
+### Toggle with image
 
-```ts file="./DropdownBadge.tsx"
-```
+The `icon` property can be used to display an image in the `DropdownToggle` and can be used alongside a text label.
 
-### Icon only
-
-```ts file="./DropdownIconOnly.tsx"
+```ts file="./DropdownImageAndText.tsx"
 ```
 
-### Split button
+### Toggle with badge
 
-```ts file="./DropdownSplitButton.tsx"
-```
+You may use a `<BadgeToggle>` to display the number of dropdown menu items.
 
-### Split button (with text)
-
-```ts file="./DropdownSplitButtonText.tsx"
+```ts file="./DropdownBadge.tsx"
 ```
 
-### Split button (indeterminate state)
+### Split button dropdown toggle
 
-```ts file="./DropdownSplitButtonIndeterminate.tsx"
-```
+Use a split button to combine an action button or other control with a dropdown menu. A `<DropdownToggle>` can be rendered as a split button by using the `splitButtonItems` property. Elements to be displayed before the toggle button must be included in `splitButtonItems`. 
 
-### Split button (disabled)
+Split buttons can utilize some of the same properties available to [button](/components/button) components, such as `isDisabled` to disable the button.
 
-```ts file="./DropdownSplitButtonDisabled.tsx"
+The following example shows a split button with a `<DropdownToggleCheckbox>`, which is primarily used for bulk select operations on large data sets. This example also includes a text label, which may be omitted.
+
+```ts file="./DropdownSplitButtonText.tsx"
 ```
 
-### Split button action
+### Split button and toggle variants
 
-```ts file="./DropdownSplitButtonAction.tsx"
-```
+Use the `toggleVariant` and `splitButtonVariant` properties to adjust the styling of split button toggles. 
 
-### Split button primary action
+The following example uses `toggleVariant="primary"` to apply primary visual styling and `splitButtonVariant="action"` to include an action.
 
 ```ts file="./DropdownSplitButtonActionPrimary.tsx"
 ```
 
-### Split button secondary action
+### Split button indeterminate state
```

Comment:
This example is not needed.  You can deletes it.  Can you also delete the `DropdownSplitButtonIndeterminate.tsx` , or open a follow up issue to dod so.